packages/web/CONTRIBUTING.md

```diff
@@ -126,6 +126,13 @@ Now use the reference from the theme in component styles:
 - `% yarn test` for test package (lint, format, unit testing, types)
 - `% yarn test:unit` for unit tests
 
+### SASS Unit Testing
+
+We use [sass-true][sass-true] for unit testing of our SASS components. The
+tests are located in `src/scss/components/<ComponentName>/__tests__` and are
+named `<ComponentName>.test.scss`. The tests are run as a part of the Jest
+testing suite. We aim to have 100% coverage of our mixins and functions.
```

Comment:
> We aim to have 100% coverage of our mixins and functions.

I think it is obvious that we want to have a `100 %` (with space) coverage. But it is embarrassing if this sentence will be here and the coverage will not be there :-/

Reply:
:D ok, I will rewrite it to highlight we focus on testing mixins and functions :) and I won't mention any percentages :D